custom_cards/custom_card_nik_clock/custom_card_nik_clock.yaml

```
@@ -0,0 +1,40 @@
+---
+custom_card_nik_clock:
+  template:
+    - "ulm_language_variables"
+  variables:
+    ulm_custom_card_nik_clock_label: "[[[ return variables.ulm_custom_card_nik_clock_label ]]]"
+    ulm_custom_card_nik_clock_switch: "[[[ return variables.ulm_custom_card_nik_clock_switch ]]]"
+    ulm_custom_card_nik_clock_switch_enable: false
+  show_icon: false
+  show_name: false
+  show_state: true
+  show_label: true
+  entity: "sensor.time"
```

Hardcoding an entity is now advised as not everyone has set up this entity.
You could use some Javascript template to extract the local time.
```suggestion
  name: >
    [[[
      const time = new Date();
      let hour = time.getHours();
      let minute = time.getMinutes();
      return hour + ":" + minute;
    ]]]
```